modules/reference/pages/rpk/rpk-cluster/rpk-cluster-partitions-move-cancel.adoc

```diff
@@ -0,0 +1,51 @@
+= rpk cluster partitions move-cancel
+:description: rpk cluster partitions move-cancel
+
+Cancel ongoing partition movements.

+

+By default, this command cancels all the partition movements in the cluster. 

+To ensure that you do not accidentally cancel all partition movements, this 

+command prompts users for confirmation before issuing the cancellation request. 

+You can use "--no-confirm" to disable the confirmation prompt:

+

+    rpk cluster partitions move-cancel --no-confirm

+

+If "--node" is set, this command will only stop the partition movements 

+occurring in the specified node:

+

+    rpk cluster partitions move-cancel --node 1
+
+== Usage
+
+[,bash]
+----
+rpk cluster partitions move-cancel [flags]
+----
+
+== Aliases
+
+[,bash]
+----
+move-cancel, alter-cancel
+----
+
+== Flags
+
+[cols="1m,1a,2a]
+|===
+|*Value* |*Type* |*Description*
+
+|`-h, --help` |- |Help for move-cancel.
+
+|`--no-confirm` |- |Disable confirmation prompt.
+
+|`--node` |int |ID of a specific node on which to cancel ongoing partition movements (default -1).
+
+|`--config` |string |Redpanda or rpk config file; default search paths are ~/.config/rpk/rpk.yaml, $PWD, and /etc/redpanda/`redpanda.yaml`.

```

Comment:
Please recheck your tags here--not rendering correctly. I think it should be:
`~/.config/rpk/rpk.yaml`, `$PWD`, and `/etc/redpanda/`redpanda.yaml`